docs/README.txt

```diff
@@ -56,4 +56,4 @@ You will need a gpu CI environment.
 Define the Order of Tutorials
 -----------------------------
 You can define the order of tutorials with `conf.py::subsection_order` and `conf.py::within_subsection_order`.
-By default, the tutorials within one subsection is sorted by filename.
\ No newline at end of file
+By default, the tutorials within one subsection is sorted by filename.
```

Comment:
Not sure what's going on with this line.

Reply:
oh i think it's a whitespace thing